debug/gdbdiff.ts

```diff
@@ -7,13 +7,27 @@
  * Copyright (C) 2021, Uri Shaked.
  **/
 
-import { GDBClient } from '../src/utils/gdbclient';
+import { GDBClient, dumpUint32, registerNames } from '../src/utils/gdbclient';
+
+function printComparedRegisters(emulator: Uint32Array, silicone: Uint32Array) {
+  console.log('Registers \t Emulator \t     Silicone');
+  for (let i = 0; i < registerNames.length; i++) {
+    console.log(
+      registerNames[i],
+      '    \t\t0x' + dumpUint32(emulator[i]) + '\t\t 0x' + dumpUint32(silicone[i])
+    );
+  }
```

Comment:
Also, it will be helpful if we print the individual APSR flag values (N/Z/C/V), that's where most of the problems are.

I think we can just print a list of the flags with are on, e.g. if only N and C are true, we'll print `[N-C-]` ,and if only Z is true we'll print `[-Z--]`, and if all flags are empty `[----]`